acceptance/tests/environment/environment_scenario-bad.rb

```diff
@@ -16,32 +16,32 @@
 apply_manifest_on(master, environment_manifest(testdir), :catch_failures => true)
```

Comment:
@er0ck I'm confused as to what is being fixed in this PR.

Is this PR intended to fix the failures we've been seeing with the `environment_scenario-non_existent.rb` test in Puppet Server, as described in PUP-5041?  Note that we saw this fail again as recently as yesterday - http://jenkins-enterprise.delivery.puppetlabs.net/view/puppet-server/view/all/job/enterprise_puppet-server_integration-system_no-conditional_full-master/99/LAYOUT=redhat6-64ma-64a,LDAP_TYPE=default,PLATFORM=default,label=beaker/testReport/junit/(root)/ruby_puppet_acceptance_tests_environment/environment_scenario_non_existent_rb/.

It had seemed that @rlinehan had pinpointed the root cause in her first comment in the PUP-5041:

> It looks like it's using "#{testdir}/environments" as the path (which is the environment path), despite the fact that the path variable is set to master.puppet('master')['codedir'].

That had seemed like a plausible explanation for the mismatched expectations for the error messages that we see when these tests are run from within the Puppet Server CI pipeline.  Is this something you are planning on including in this PR or is there a separate PR coming for that?